CONTIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing
+
+First off, thank you for considering contributing to this project! Now, to start contributing:
+
+## Issues and suggestions
+
+If you either find a bug or have any suggestion or opinion you want to discuss and share, please open an issue and add the proper label to it so we can get in contact with you.
+There are no wrong opinions! All feedback is welcome to make this the most suitable tool for you to use and for us to grow.
+
+## How to contribute
+
+If you have a new feature you want to add or a bug you think you can fix, follow this steps:
+
+1. Fork the repo
+2. Create your feature branch (`git checkout -b my-new-feature`)
+3. Commit your changes (`git commit -am 'Add some feature'`)
+4. Push to the branch (`git push origin my-new-feature`)
+5. Create new Pull Request with **clear title and description**
+
+##  Testing
+
+Your new feature **must** be tested with the proper tools. In this project, we use Jest. Once your tests are written, run:
+
```

Comment:
I think that it'd be nice to add that the tests will fail if the `TEST: INITIALIZE - Initialize Firestore Service` is marked as `.skip()` (true story)

Reply:
can you elaborate a replacement text for this?

Reply:
something along the lines of `TEST: INITIALIZE - Initialize Firestore Service` must not be marked as `skipped` for the tests to run properly. This means that if you want to run some test individually. `TEST: INITIALIZE` and your chosen test must not be `skipped`

Reply:
thank youuuu 👍